config/config.go

```diff
@@ -128,16 +130,17 @@ func Get() *ExportConfig {
 		kubenv.AutomaticEnv()
 
 		config = &ExportConfig{
-			Hostname:           kubenv.GetString("Hostname"),
-			PublicPort:         options.GetInt("PUBLIC_PORT"),
-			MetricsPort:        options.GetInt("METRICS_PORT"),
-			PrivatePort:        options.GetInt("PRIVATE_PORT"),
-			Debug:              options.GetBool("DEBUG"),
-			LogLevel:           options.GetString("LOG_LEVEL"),
-			OpenAPIPublicPath:  options.GetString("OPEN_API_FILE_PATH"),
-			OpenAPIPrivatePath: options.GetString("OPEN_API_PRIVATE_PATH"),
-			Psks:               options.GetStringSlice("PSKS"),
-			ExportExpiryDays:   options.GetInt("EXPORT_EXPIRY_DAYS"),
+			Hostname:           	kubenv.GetString("Hostname"),
+			PublicPort:         	options.GetInt("PUBLIC_PORT"),
+			MetricsPort:        	options.GetInt("METRICS_PORT"),
+			PrivatePort:        	options.GetInt("PRIVATE_PORT"),
+			Debug:              	options.GetBool("DEBUG"),
+			LogLevel:           	options.GetString("LOG_LEVEL"),
+			OpenAPIPublicPath:  	options.GetString("OPEN_API_FILE_PATH"),
+			OpenAPIPrivatePath: 	options.GetString("OPEN_API_PRIVATE_PATH"),
+			Psks:               	options.GetStringSlice("PSKS"),
+			ExportExpiryDays:   	options.GetInt("EXPORT_EXPIRY_DAYS"),
+			ExportableApplications: options.GetStringMapStringSlice("EXPORT_ENABLE_APPS"),
```

Comment:
I would perform the conversion from `map[string][]string` to `map[string]map[string]bool` right here.  That way we just have to do it one time and then the code in the `exports` module just has to check it.

Something like this should work:

```
var exportableAppsFromConfig map[string][]string
ExportableApplications: convertExportableAppsFromConfigToInternal(options.GetStringMapStringSlice("EXPORT_ENABLE_APPS")),
```